doc/thing/thing-requirements.rst

```diff
@@ -12,125 +12,184 @@ On other systems, you may consider using the `KNoT Docker <thing-docker.html>`_
 
-.. note:: If the system can't find west, try logging out and in again.
+   .. code-block:: bash
 
-- Set up zephyr environment variables
+      wget -O $HOME/Downloads/zephyr-sdk-0.10.0-setup.run https://github.com/zephyrproject-rtos/sdk-ng/releases/download/v0.10.0/zephyr-sdk-0.10.0-setup.run && \
+           chmod +x $HOME/Downloads/zephyr-sdk-0.10.0-setup.run  && \
+           $HOME/Downloads/zephyr-sdk-0.10.0-setup.run -- -d ~/zephyr-sdk-0.10.0
 
-.. code-block:: bash
+#. Program the ``$HOME/.profile`` to always set **ZEPHYR_TOOLCHAIN_VARIANT** and **ZEPHYR_SDK_INSTALL_DIR**.
 
-   $ source zephyr/zephyr-env.sh
+   .. code-block:: bash
 
-- Program the ``$HOME/.profile`` to always source zephyr-env.sh when you log in.
+      echo "export ZEPHYR_TOOLCHAIN_VARIANT=zephyr" >> $HOME/.profile
+      echo "export ZEPHYR_SDK_INSTALL_DIR=$HOME/zephyr-sdk-0.10.0" >> $HOME/.profile
 
-.. code-block:: bash
+Init Zephyr Environment
+'''''''''''''''''''''''
 
-   $ echo "source $ZEPHYR_BASE/zephyr-env.sh" >> $HOME/.profile
+#. If you do not have ``~/.local/bin`` on your PATH environment variable, add it.
 
-.. note:: If you skip this step, it will be necessary to manually source zephyr-env.sh every time a new terminal is opened.
+   .. code-block:: bash
+
+      echo PATH=\"$HOME/.local/bin:'$PATH'\" >> $HOME/.profile
+      source $HOME/.profile
+
+#. Install the west binary and bootstrapper.
+
+   .. code-block:: bash
+
+      pip3 install --user west
+
+#. Clone KNoT Zephyr fork.
+
+   .. code-block:: bash
+
+      git clone -b zephyr-knot-v1.14.0 https://github.com/CESARBR/zephyr.git $HOME/zephyrproject/zephyr/
+
+   .. note:: It will create a folder under $HOME directory and clone zephyr inside it. Make sure to update the path on the following steps if you clone it under another folder.
+
+#. Initialize west.
+
+   .. code-block:: bash
+
+      cd $HOME/zephyrproject/
+      west init -l zephyr/
+      west update
+
+   .. note:: If the system can't find west, try logging out and in again.
+
+#. Set up zephyr environment variables.
+
+   .. code-block:: bash
+
+      source $HOME/zephyrproject/zephyr/zephyr-env.sh
+
+#. Program the ``$HOME/.profile`` to always source zephyr-env.sh when you log in.
+
+   .. code-block:: bash
+
+      echo "source $HOME/zephyrproject/zephyr/zephyr-env.sh" >> $HOME/.profile
+
+   .. note:: If you skip this step, it will be necessary to manually source zephyr-env.sh every time a new terminal is opened.
 
 ----------------------------------------------------------------
 
 nRF5x Command Line Tools and Segger JLink
 -----------------------------------------
 
-- Download and extract cli applications at `nRF5 Command Line Tools <https://www.nordicsemi.com/Software-and-Tools/Development-Tools/nRF5-Command-Line-Tools>`_.
+Download and extract cli applications at `nRF5 Command Line Tools <https://www.nordicsemi.com/Software-and-Tools/Development-Tools/nRF-Command-Line-Tools>`_.
```

Comment:
Make it clear that you can download and extract using the links or following the below steps.